Add seo meta tags for marketplace

glespinosa opened the pull request:
Add seo tags title,description,image

glespinosa committed:
Add seo meta tags for marketplace

Add seo tags title,description,image

ardeay commented on src/pages/marketplace/index.js:
```diff
@@ -14,9 +15,13 @@ const Marketplace = ({
+      <Head>
+        <title>{props.title}</title>
+        <meta property="og:title" content={props.title} />
+        <meta property="og:description" content={props.description} />
```
Each toJSON object has a meta property, which contains actual meta for meta description and meta title. Please inspect the incoming object and use that. For example `props.meta.web. seo_meta_description `
![image](https://user-images.githubusercontent.com/729972/169875143-9700ef89-de13-432d-9403-17553250e750.png)

glespinosa committed:
Get title,description to meta.web.

Get title,description to meta.web and remove unnessary props passed in Main component.
